spanner/api/Spanner.Samples/ListInstanceConfigsAsync.cs

```diff
@@ -19,12 +19,13 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
+using System.Threading.Tasks;
 
-public class ListInstanceConfigsSample
+public class ListInstanceConfigsAsyncSample
```

Comment:
Changing to creating the client async is fine, but you cannot change the file name because the existing sample was already included in docs.

Either we keep the file name incosistent (I'm fine with that in this instance) or you coordinate with the Spanner technical writers to change the docs to start using the new file name.